docs/reference/src/language/concepts/translations.md

```diff
@@ -75,99 +92,110 @@ export component MenuItem {
 
-The `.po` file will contain the strings in a target language.
+The `.po` file contains the strings in a target language.
 
-`.po` and `.pot` files are plain text files, that you can edit with a text editor. We recommend
+`.po` and `.pot` files are plain text files that you can edit with a text editor. We recommend
 using a dedicated translation tool for working with them, such as the following:
 
- - [poedit](https://poedit.net/)
- - [OmegaT](https://omegat.org/)
- - [Lokalize](https://userbase.kde.org/Lokalize)
- - [Transifex](https://www.transifex.com/) (web interface)
+-   [poedit](https://poedit.net/)
+-   [OmegaT](https://omegat.org/)
+-   [Lokalize](https://userbase.kde.org/Lokalize)
+-   [Transifex](https://www.transifex.com/) (web interface)
 
 ## Convert `.po` Files to `.mo` Files
 
-The human readable `.po` files need to be converted into machine-friendly `.mo` files, a binary representation
-that is very efficient to read.
+Convert the human readable `.po` files into machine-friendly `.mo` files, which are a binary representation
+that is more efficient to read by code.
 
 Use [Gettext](https://www.gnu.org/software/gettext/)'s `msgfmt` command line tool to convert `.po` files to `.mo`
 files:
 
-```
+```sh
 msgfmt translation.po -o translation.mo
 ```
 
 ## Select and Load `.mo` Files at Run-Time
 
 Slint uses the [Gettext](https://www.gnu.org/software/gettext/) library to load translations at run-time.
-Gettext locates the translation file in the following location:
-Gettext expects translation files - called message catalogs - to be placed in following directory hierarchy:
+Gettext expects translation files - called message catalogs - in following directory hierarchy:
 
 ```
 dir_name/locale/LC_MESSAGES/domain_name.mo
 ```
 
-* `dir_name`: the base directory that you can choose freely.
-* `locale`: The name of the user's locale for a given target language, such as `fr` for French, or `de` for German.
-  The locale is typically determined using environment variables that your operating system sets.
-* `domain_name`: Selected based on the programming language you're using Slint with.
+-   `dir_name`: the base directory that you can choose freely.
+-   `locale`: The name of the user's locale for a given target language, such as `fr` for French, or `de` for German.
+
+    The locale is typically determined using environment variables that your operating system sets.
+
+-   `domain_name`: Selected based on the programming language you're using Slint with.
 
-For more info, see the [Gettext documentation](https://www.gnu.org/software/gettext/manual/gettext.html#Locating-Catalogs).
+:::info
+
+Read the [Gettext documentation](https://www.gnu.org/software/gettext/manual/gettext.html#Locating-Catalogs) for more information.
+
+:::
 
 ### Select and Load Translations with Rust
 
-First, enable the `gettext` feature of the `slint` create to gain access to the translations API
+<!-- TODO: Do people know how to do this? -->
```

Comment:
For rust I think so, yes. Adding to `features` in their `Cargo.toml`.

Reply:
Ok, added a small clarification in case